src/Sylius/Bundle/ApiBundle/Resources/config/routing/customer.yml

```diff
@@ -1,8 +1,43 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+sylius_api_customer_index:
+    path: /
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.customer:indexAction
+        _sylius:
+            serialization_groups: [Default]
```

Comment:
No, it doesn't. `ResourceController` doesn't use any serialization group [by default](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php#L418)
Removing this group will result in following diff:
```console
     "_embedded": {
         "items": [
             {
-                "id": @integer@,
+                "id": 6116,
                 "email": "Joe@doe.com",
+                "email_canonical": "joe@doe.com",
                 "first_name": "Joe",
-                "last_name": "Doe"
+                "last_name": "Doe",
+                "birthday": "1996-11-24T00:00:00+0100",
+                "gender": "u",
+                "groups": []
             },
             {
-                "id": @integer@,
+                "id": 6115,
                 "email": "Barry@doe.com",
+                "email_canonical": "barry@doe.com",
                 "first_name": "Barry",
-                "last_name": "Doe"
+                "last_name": "Doe",
+                "birthday": "1988-12-16T00:00:00+0100",
+                "gender": "u",
+                "groups": []
             },
             {
-                "id": @integer@,
+                "id": 6114,
                 "user": {
-                    "id": @integer@,
+                    "id": 4222,
                     "username": "Thea@doe.com",
+                    "username_canonical": "thea@doe.com",
+                    "roles": [
+                        "ROLE_USER"
+                    ],
                     "enabled": true
                 },
                 "email": "Thea@doe.com",
+                "email_canonical": "thea@doe.com",
                 "first_name": "Thea",
-                "last_name": "Doe"
+                "last_name": "Doe",
+                "birthday": "2011-09-25T00:00:00+0200",
+                "gender": "u",
+                "groups": []
             },
             {
-                "id": @integer@,
+                "id": 6113,
                 "user": {
-                    "id": @integer@,
+                    "id": 4221,
                     "username": "Roy@doe.com",
+                    "username_canonical": "roy@doe.com",
+                    "roles": [
+                        "ROLE_USER"
+                    ],
                     "enabled": true
                 },
                 "email": "Roy@doe.com",
+                "email_canonical": "roy@doe.com",
                 "first_name": "Roy",
-                "last_name": "Doe"
+                "last_name": "Doe",
+                "birthday": "2000-11-30T00:00:00+0100",
+                "gender": "u",
+                "groups": []
             },
             {
-                "id": @integer@,
+                "id": 6112,
                 "user": {
-                    "id": @integer@,
+                    "id": 4220,
                     "username": "Oliver@doe.com",
+                    "username_canonical": "oliver@doe.com",
+                    "roles": [
+                        "ROLE_USER"
+                    ],
                     "enabled": true
                 },
                 "email": "Oliver@doe.com",
+                "email_canonical": "oliver@doe.com",
                 "first_name": "Oliver",
-                "last_name": "Doe"
+                "last_name": "Doe",
+                "birthday": "1975-06-20T00:00:00+0100",
+                "gender": "u",
+                "groups": []
             },
             {
-                "id": @integer@,
+                "id": 6111,
                 "user": {
-                    "id": @integer@,
+                    "id": 4219,
                     "username": "api@sylius.com",
+                    "roles": [
+                        "ROLE_USER",
+                        "ROLE_API"
+                    ],
                     "enabled": true
                 },
-                "email": "api@sylius.com"
+                "email": "api@sylius.com",
+                "email_canonical": "api@sylius.com",
+                "gender": "u",
+                "groups": []
             }
         ]
     }
 }
-

```